front-end/src/Routes/HomeRoute.js

```diff
@@ -1,19 +1,19 @@
 import React from "react";
-import Header from "../utils/Header";
+import { Redirect } from "react-router";
 import NumberContainer from "../Numbers/NumberContainer";
+import NotifyHeader from "../utils/NotifyHeader";
 import NumberListFetcher from "../Numbers/NumberListFetcher";
 import { AppContainer } from "../utils/styles";
 
-export default () => (
-  <AppContainer>
+export default ({ location }) => (
+  <AppContainer emptySize="70px">
+    {location.search.includes("tv") && (
+      <Redirect to="/tv" />
+    ) /*Any url with TV in it redirects to tv*/}
     <NumberListFetcher
       render={({ numberList }) => [
-        <Header key="header" />,
-        <NumberContainer
-          numberList={numberList}
-          isAdmin={false}
-          key="numbers"
-        />
+        <NotifyHeader numberList={numberList} />,
```

Comment:
Are keys not required for React to be able to quickly differentiate between the elements when diffing? IIRC React usually yells at you if you don't set `key` when returning an array of elements..

Reply:
Yes but only on tags that are duplicated need it, there is only one ```<NotifyHeader />```, but in:
```javascript
["a", "b", "c"].map(letter => <div>{letter}</div>)
```
the div needs a key since all the divs in the parent components look the same. By default the index from the map is used, which will cause unwanted behaviour when removing items from the list. (hope i was not over explaining :) )

Reply:
Oh right, makes sense :) Thanks!